server/controllers/finance/fiscal.js

```diff
@@ -126,12 +136,24 @@ function create(req, res, next) {
   record.user_id = req.session.user.id;
   record.enterprise_id = req.session.enterprise.id;
   record.start_date = new Date(record.start_date);
+  record.end_date = new Date(record.end_date);
+
+  let params = [
+    record.enterprise_id, record.previous_fiscal_year_id, record.user_id,
+    record.label, record.number_of_months,
+    record.start_date, record.end_date, record.note
+  ];
 
-  const sql = 'INSERT INTO fiscal_year SET ?;';
+  let transaction = new Transaction(db);
 
-  db.exec(sql, [record])
-    .then(function (result) {
-      res.status(201).json({ id : result.insertId });
+  transaction
+    .addQuery('SET @fiscalYearId = 0;')
```

Comment:
Hmm...  I don't have any good ideas to prevent it.

Since Fiscal Year isn't going to be used much (~1 times a year?), I'll merge this in without change.  Could you make an issue requesting a fix the `@fiscalYearId`?  That way we can make sure to revisit this in the future.